examples/msbr/openmc_msbr_model.py

```diff
@@ -0,0 +1,1045 @@
+import openmc
+import numpy as np
+import argparse
+
+from openmc.deplete import CoupledOperator, PredictorIntegrator, CELIIntegrator
+
+from _core_elements import *
+from _control_rods import *
+from _root_geometry import *
```

Comment:
```suggestion
import _core_elements
import _control_rods
import _root_geometry
```

I'm going back and forth on the right "solution" to this. In general it's not good to `import *`, so that should change. It's up to you if you want to import every function, or just use the module name space (e.g. `import numpy as np` vs `from numpy import *`).

Reply:
It's also not clear to me why these modules are "private."

Reply:
I will clean this up.